youtrack_sdk/client.py

```diff
@@ -430,6 +463,28 @@ def get_projects(self, offset: int = 0, count: int = -1) -> Sequence[Project]:
             ),
         )
 
+    def get_project_work_item_types(
+        self,
+        *,
+        project_id: str,
+        offset: int = 0,
+        count: int = -1,
```

Comment:
Actually, I wonder if this is really works, or it does return 42 implicitly and nobody has noticed it so far:

https://www.jetbrains.com/help/youtrack/devportal/api-concept-pagination.html#activities-pagination

Using negative values is not documented anywhere to the best of my knowledge. Also, from typing perspective we should have rather done something like `count: Optional[int] = None` and hidden then `-1` trick inside of `_build_url`.

But this can be a topic of a different PR.

Reply:
I tried to get a list of issues on our test server with `count=-1` and got 294 entries, which is the correct value. But I agree with you. It's better to replace `offset` and `count` type with the `Optional[int]`. Especially since `_build_url` already accepts them as `Optional[int]` and removes from URL if they're set to `None`. I'll create a PR for this.

Reply:
I tested `get_issues` on another server and `count=-1` seems to work. Server returns 1000 entries when I set `count=None` and all entries if I set `count=-1`.

Reply:
https://github.com/moneymeets/youtrack-sdk/pull/45